### DISCUSSION TITLE
Multiple Authentication

### DISCUSSION THREAD

**Discussion by keskinadem** [strongswan/strongswan#1894] [General]
What is difference between N(MULT_AUTH) and N(AUTH_FOLLOWS) and why MULT_AUTH is negotiated in second and third messages instead of IKE_SA_INIT(first&second messages) as usual.

**Comment by tobiasbrunner** [comment]
Please refer to [RFC 4739](https://datatracker.ietf.org/doc/html/rfc4739).
